classes/Http/Controller/Reviewer/TalksController.php

```diff
@@ -35,25 +35,6 @@ public function __construct(TalkHandler $talkHandler, Twig_Environment $twig, Ur
         parent::__construct($twig, $urlGenerator);
     }
 
-    public function viewAction(Request $request): Response
-    {
-        $this->talkHandler->grabTalk((int) $request->get('id'));
-
-        if (!$this->talkHandler->view()) {
-            $request->getSession()->set('flash', [
-                'type'  => 'error',
-                'short' => 'Error',
-                'ext'   => 'Could not find requested talk',
-            ]);
-
-            return $this->redirectTo('admin_talks');
```

Comment:
Note how there's a mistake here, as the route `admin_talks` should be `reviewer_talks`.